### ISSUE TITLE
Diffs in tests when all packages are loaded

### ISSUE THREAD

**Issue by olexandr-konovalov** [gap-packages/Memoisation#7]
#5 is fixed, but now this: https://travis-ci.org/gap-infra/gap-docker-pkg-tests-stable-4.10-devel/jobs/574962703

```
########> Diff in /home/gap/Memoisation/gaproot/pkg/Memoisation/tst/DiskCache.\
tst:80
# Input is:
exp(2.0);;  # same key as last
# Expected output:
#I  Memo key: 2.
#I  Key known!  Loading result from cache...
#I  Using filename MEMODIR/exp/e_to_the_2..out
#I  Got 25 bytes from file
#I  Got cached result from file
# But found:
#I  Memo key: 2.
#I  Key known!  Loading result from cache...
#I  Using filename /tmp/tmZRIMqx/exp/e_to_the_2..out
#I  Got 37 bytes from file
#I  Got cached result from file
########
```

**Comment by olexandr-konovalov** [comment]
This is in stable-4.10 branch of GAP when all packages are loaded.

**Comment by olexandr-konovalov** [comment]
Same in the GAP master branch https://travis-ci.org/gap-infra/gap-docker-pkg-tests-master-devel/jobs/574962887

**Comment by mtorpey** [comment]
Thanks for this.  It's probably caused by a different `IO_Pickle` method, or a different result, for floats.  The tests should probably be changed to be more accommodating (the number of bytes in a file is not important), but I'll investigate to make sure that this wouldn't cause bugs.

**Comment by olexandr-konovalov** [comment]
Please note also "Using filename..." diff.

**Comment by mtorpey** [comment]
The "Using filename..." diff actually isn't a problem – I've got a custom `compareFunction` that matches `MEMODIR` to whatever temporary directory is being used.  So I expect this to pass after the other part is resolved. :)

**Comment by mtorpey** [comment]
The diff is caused by a different `IO_Pickle` method being used when the `float` package is loaded: `float-0.9.1/lib/pickle.g:3`.  This isn't a problem, and it's about time I lowered the infolevel of the package tests anyway.

**Comment by mtorpey** [comment]
Hopefully this is fixed by c25d769.
